controllers/flashsystemcluster_controller.go

```diff
@@ -285,6 +268,12 @@ func (r *FlashSystemClusterReconciler) reconcile(instance *odfv1alpha1.FlashSyst
 		UID:        instance.UID,
 	}
 
+	newCMOwnerDetails, err := r.getCmOwnerDetails(instance.Namespace)
+	if err != nil {
+		r.Log.Error(err, "failed to get operator pod name", "ConfigMap", util.FscCmName)
```

Comment:
not sure if it's the correct msg, as the `getCmOwnerDetails` function can return an error in case it cannot read the operator pod object as well.
maybe something more general?

Reply:
changed